week-2/F-logical-operators/exercise2.js

```diff
@@ -5,17 +5,31 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
-
+function isNegative(num) {
+  if (num < 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+function isBetween5and10(num) {
+  if (num >= 5 && num <= 10) return true;
+}
+function isLongName(name) {
+  if (name.length > 3) return true;
+}
+function startsWithD(name) {
+  if (name.charAt(0) == "D") return true;
```

Comment:
Nice! But check what happens when you do `startsWithD("daniel")` - the string still starts with the letter `d`! :)

Reply:
yes there should be strict equality there i.e. name.charAt(0) ==="D"